echopype/utils/misc.py

```diff
@@ -11,3 +19,66 @@ def camelcase2snakecase(camel_case_str):
             camel_case_str = camel_case_str[:i] + "_" + camel_case_str[i:]
 
     return camel_case_str.lower()
+
+
+def depth_from_pressure(
+    pressure: Union[float, FloatSequence],
+    latitude: Optional[Union[float, FloatSequence]] = 30.0,
+    atm_pres_surf: Optional[Union[float, FloatSequence]] = 0.0,
+) -> NDArray[float]:
+    """
+    Convert pressure to depth using UNESCO 1983 algorithm.
+
+    UNESCO. 1983. Algorithms for computation of fundamental properties of seawater (Pressure to
+    Depth conversion, pages 25-27). Prepared by Fofonoff, N.P. and Millard, R.C. UNESCO technical
+    papers in marine science, 44. http://unesdoc.unesco.org/images/0005/000598/059832eb.pdf
+
+    Parameters
+    ----------
+    pressure : Union[float, FloatSequence]
+        Pressure in dbar
+    latitude : Union[float, FloatSequence], default=30.0
+        Latitude in decimal degrees.
+    atm_pres_surf : Union[float, FloatSequence], default=0.0
+        Atmospheric pressure at the surface in dbar
```

Comment:
```suggestion
    atm_pres_surf : Union[float, FloatSequence], default=0.0
        Atmospheric pressure at the surface in dbar.
        Use the default 0.0 value if pressure is corrected to be 0 at the surface.
        Otherwise, enter a correction for pressure due to air, sea ice and any other 
        medium that may be present
```

Reply:
I'll merge this commit suggestion myself.